server/views/embed.py

```diff
@@ -0,0 +1,27 @@
+from django.template.response import TemplateResponse
+from rest_framework.renderers import JSONRenderer
+from server.models import WfModule
+from server.serializers import WorkflowSerializer, WfModuleSerializer
+
+def embed(request, wfmodule_id):
+    try:
```

Comment:
this can be shortened to get_object_or_404

Reply:
No it can't for two reasons: 1) Django doesn't support per-view 404 pages using get_object_or_404, if you want a custom 404 you have to do it manually. Also, this doesn't send a 404. It sets wf_module and workflow to 'none' and renders the same template, so we can handle both botched embed codes and embed codes that used to work but have been made private.